torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -135,12 +135,12 @@ class PytorchBotHandler {
     } else {
       await this.handleConfused(
         true,
-        "You need to provide a reason for using force merge, in the format `@pytorchbot merge -f '[CATEGORY] Explanation'`. " +
-          "With [CATEGORY] being one the following:\n" +
-          " EMERGENCY - an emergency fix to quickly address an issue\n" +
-          " MINOR - a minor fix such as cleaning locally unused variables, which shouldn't break anything\n" +
-          " PRE_TESTED - a previous CI run tested everything and you've only added minor changes like fixing lint\n" +
-          " OTHER - something not covered above"
+        "You need to provide a reason for using force merge, in the format `@pytorchbot merge -f 'Explanation'`. " +
+          "The explanation needs to be clear on why this is needed. Here are some good examples:\n" +
```

Comment:
nit you can use the ` quotes to make it multiline by default and not have to do the \n stuff